demos/smart_classroom_demo/cpp/src/cnn.cpp

```diff
@@ -13,68 +13,74 @@
 
 #include <inference_engine.hpp>
 
+#include <map>
+
+#include "openvino/core/layout.hpp"
+#include "openvino/openvino.hpp"
+
 CnnDLSDKBase::CnnDLSDKBase(const Config& config) : config_(config) {}
 
 void CnnDLSDKBase::Load() {
-    auto cnnNetwork = config_.ie.ReadNetwork(config_.path_to_model);
-
+    auto cnnNetwork = config_.ie.read_model(config_.path_to_model);
 
-    const int currentBatchSize = cnnNetwork.getBatchSize();
-    if (currentBatchSize != config_.max_batch_size)
-        cnnNetwork.setBatchSize(config_.max_batch_size);
+    const int currentBatchSize = cnnNetwork->input().get_shape()[0];
+    if (currentBatchSize != config_.max_batch_size) {
+      cnnNetwork->get_parameters()[0]->set_layout("NCHW");
+      ov::set_batch(cnnNetwork, config_.max_batch_size);
+    }
 
-    InferenceEngine::InputsDataMap in = cnnNetwork.getInputsInfo();
+    ov::OutputVector in = cnnNetwork->inputs();
     if (in.size() != 1) {
         throw std::runtime_error("Network should have only one input");
     }
-    in.begin()->second->setPrecision(InferenceEngine::Precision::U8);
-    in.begin()->second->setLayout(InferenceEngine::Layout::NCHW);
-    input_blob_name_ = in.begin()->first;
-
-    InferenceEngine::OutputsDataMap out = cnnNetwork.getOutputsInfo();
-    for (auto&& item : out) {
-        item.second->setPrecision(InferenceEngine::Precision::FP32);
-        output_blobs_names_.push_back(item.first);
+
+    ov::preprocess::PrePostProcessor proc(cnnNetwork);
```

Comment:
I suggest to name the object `ppp` how it is done for samples

Reply:
Done